### ISSUE TITLE
Object of class Symfony\Component\VarDumper\Cloner\Stub could not be converted to string

### ISSUE THREAD

**Issue by mattwoo** [Codeception/Codeception#4322]
#### What are you trying to achieve?
Run functional tests with symfony module
#### What do you get instead?
Error 
```
PHP Fatal error:  Object of class Symfony\Component\VarDumper\Cloner\Stub could not be converted to string in /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Module/Symfony.php on line 500
```
> Provide console output if related. Use `-vvv` mode for more details.

```bash
 php vendor/bin/codecept run functional -c src/Foo/xxxBundle/ -vvv
Codeception PHP Testing Framework v2.3.3
Powered by PHPUnit 5.7.20 by Sebastian Bergmann and contributors.

Foo\XxxBundle.functional Tests (2) --------------------------------------------------------------------------------------------------------------------------------------------
Modules: Symfony
-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
Testing Foo\XxxBundle.functional
CaptchaCest: Redirects when captcha correct
Signature: Foo\XxxBundle\CaptchaCest:redirectsWhenCaptchaCorrect
Test: tests/functional/CaptchaCest.php:redirectsWhenCaptchaCorrect
Scenario --
 I am authenticated as "admin",["ROLE_ADMIN"]
   I grab service "app.jwtservice"
   I grab service "test.client.cookiejar"
   I set cookie "token","eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyb2xlcyI6WyJST0xFX0FETUlOIl0sInVzZXJuYW1lIjoiYWRtaW4iLCJleHAiOjE0OTcwMTg2NTd9.6G6mjK2sJ4FmurmhpOd3tRKHCf2aAlZU..."
  [Cookie Jar] ["token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyb2xlcyI6WyJST0xFX0FETUlOIl0sInVzZXJuYW1lIjoiYWRtaW4iLCJleHAiOjE0OTcwMTg2NTd9.6G6mjK2sJ4FmurmhpOd3tRKHCf2aAlZUIzS6RWZghJg; path=/; httponly"]
 I am on page "/sf/captcha/verify?rd=%2Fsf%2Fcaptcha%2Fverify%3Frd%3Dasdqwe"
  [Request Headers] []
  [Page] /sf/captcha/verify?rd=%2Fsf%2Fcaptcha%2Fverify%3Frd%3Dasdqwe
  [Response] 200
  [Request Cookies] {"token":"eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJyb2xlcyI6WyJST0xFX0FETUlOIl0sInVzZXJuYW1lIjoiYWRtaW4iLCJleHAiOjE0OTcwMTg2NTd9.6G6mjK2sJ4FmurmhpOd3tRKHCf2aAlZUIzS6RWZghJg"}
  [Response Headers] {"cache-control":["no-cache, private"],"date":["Fri, 09 Jun 2017 13:30:57 GMT"],"content-type":["text/html; charset=UTF-8"],"x-debug-token":["94f4dc"],"set-cookie":["MOCKSESSID=cb2f7618d034c72ea9ce17ccefafffdb0eb7d25957ac81213a1a0c4acdbff2fc; path=/"]}
PHP Warning:  Uncaught PHPUnit_Framework_Exception: Object of class Symfony\Component\VarDumper\Cloner\Stub could not be converted to string

/var/www/html/symfony/src/Foo/XxxBundle/tests/_support/_generated/FunctionalTesterActions.php:334
/var/www/html/symfony/src/Foo/XxxBundle/tests/functional/CaptchaCest.php:15

  thrown in /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Subscriber/ErrorHandler.php on line 75
PHP Stack trace:
PHP   1. {main}() /var/www/html/symfony/vendor/codeception/codeception/codecept:0
PHP   2. Codeception\Application->run() /var/www/html/symfony/vendor/codeception/codeception/codecept:36
PHP   3. Symfony\Component\Console\Application->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Application.php:103
PHP   4. Symfony\Component\Console\Application->doRun() /var/www/html/symfony/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:130
PHP   5. Symfony\Component\Console\Application->doRunCommand() /var/www/html/symfony/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:223
PHP   6. Symfony\Component\Console\Command\Command->run() /var/www/html/symfony/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:869
PHP   7. Codeception\Command\Run->execute() /var/www/html/symfony/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:264
PHP   8. Codeception\Command\Run->runSuites() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Command/Run.php:307
PHP   9. Codeception\Codecept->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Command/Run.php:380
PHP  10. Codeception\Codecept->runSuite() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Codecept.php:153
PHP  11. Codeception\SuiteManager->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Codecept.php:184
PHP  12. Codeception\PHPUnit\Runner->doEnhancedRun() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/SuiteManager.php:157
PHP  13. PHPUnit_Framework_TestSuite->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/PHPUnit/Runner.php:106
PHP  14. Codeception\Test\Test->run() /var/www/html/symfony/vendor/phpunit/phpunit/src/Framework/TestSuite.php:722
PHP  15. Codeception\Test\Cest->test() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Test/Test.php:87
PHP  16. Codeception\Test\Cest->executeTestMethod() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Test/Cest.php:82
PHP  17. Codeception\Test\Cest->invoke() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Test/Cest.php:148
PHP  18. Codeception\Lib\Di->injectDependencies() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Test/Cest.php:136
PHP  19. ReflectionMethod->invokeArgs() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/Di.php:127
PHP  20. Foo\XxxBundle\CaptchaCest->redirectsWhenCaptchaCorrect() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/Di.php:127
PHP  21. Foo\XxxBundle\FunctionalTester->amOnPage() /var/www/html/symfony/src/Foo\XxxBundle/tests/functional/CaptchaCest.php:15
PHP  22. Codeception\Scenario->runStep() /var/www/html/symfony/src/Foo\XxxBundle/tests/_support/_generated/FunctionalTesterActions.php:334
PHP  23. Codeception\Step->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Scenario.php:72
PHP  24. call_user_func_array:{/var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Step.php:260}() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Step.php:260
PHP  25. Codeception\Lib\InnerBrowser->amOnPage() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Step.php:260
PHP  26. Codeception\Lib\InnerBrowser->_loadPage() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/InnerBrowser.php:331
PHP  27. Codeception\Lib\InnerBrowser->clientRequest() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/InnerBrowser.php:239
PHP  28. Codeception\Module\Symfony->debugResponse() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/InnerBrowser.php:192
PHP  29. implode() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Module/Symfony.php:500
PHP Fatal error:  Object of class Symfony\Component\VarDumper\Cloner\Stub could not be converted to string in /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Module/Symfony.php on line 500
PHP Stack trace:
PHP   1. {main}() /var/www/html/symfony/vendor/codeception/codeception/codecept:0
PHP   2. Codeception\Application->run() /var/www/html/symfony/vendor/codeception/codeception/codecept:36
PHP   3. Symfony\Component\Console\Application->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Application.php:103
PHP   4. Symfony\Component\Console\Application->doRun() /var/www/html/symfony/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:130
PHP   5. Symfony\Component\Console\Application->doRunCommand() /var/www/html/symfony/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:223
PHP   6. Symfony\Component\Console\Command\Command->run() /var/www/html/symfony/vendor/symfony/symfony/src/Symfony/Component/Console/Application.php:869
PHP   7. Codeception\Command\Run->execute() /var/www/html/symfony/vendor/symfony/symfony/src/Symfony/Component/Console/Command/Command.php:264
PHP   8. Codeception\Command\Run->runSuites() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Command/Run.php:307
PHP   9. Codeception\Codecept->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Command/Run.php:380
PHP  10. Codeception\Codecept->runSuite() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Codecept.php:153
PHP  11. Codeception\SuiteManager->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Codecept.php:184
PHP  12. Codeception\PHPUnit\Runner->doEnhancedRun() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/SuiteManager.php:157
PHP  13. PHPUnit_Framework_TestSuite->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/PHPUnit/Runner.php:106
PHP  14. Codeception\Test\Test->run() /var/www/html/symfony/vendor/phpunit/phpunit/src/Framework/TestSuite.php:722
PHP  15. Codeception\Test\Cest->test() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Test/Test.php:87
PHP  16. Codeception\Test\Cest->executeTestMethod() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Test/Cest.php:82
PHP  17. Codeception\Test\Cest->invoke() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Test/Cest.php:148
PHP  18. Codeception\Lib\Di->injectDependencies() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Test/Cest.php:136
PHP  19. ReflectionMethod->invokeArgs() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/Di.php:127
PHP  20. Foo\XxxBundle/CaptchaCest->redirectsWhenCaptchaCorrect() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/Di.php:127
PHP  21. Foo\XxxBundle\FunctionalTester->amOnPage() /var/www/html/symfony/src/Foo/XxxBundle/tests/functional/CaptchaCest.php:15
PHP  22. Codeception\Scenario->runStep() /var/www/html/symfony/src/Foo/XxxBundle/tests/_support/_generated/FunctionalTesterActions.php:334
PHP  23. Codeception\Step->run() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Scenario.php:72
PHP  24. call_user_func_array:{/var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Step.php:260}() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Step.php:260
PHP  25. Codeception\Lib\InnerBrowser->amOnPage() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Step.php:260
PHP  26. Codeception\Lib\InnerBrowser->_loadPage() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/InnerBrowser.php:331
PHP  27. Codeception\Lib\InnerBrowser->clientRequest() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/InnerBrowser.php:239
PHP  28. Codeception\Module\Symfony->debugResponse() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Lib/InnerBrowser.php:192
PHP  29. implode() /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Module/Symfony.php:500



FATAL ERROR. TESTS NOT FINISHED.
Object of class Symfony\Component\VarDumper\Cloner\Stub could not be converted to string
in /var/www/html/symfony/vendor/codeception/codeception/src/Codeception/Module/Symfony.php:500

Remaining deprecation notices (5)

The Symfony\Component\EventDispatcher\ContainerAwareEventDispatcher class is deprecated since version 3.3 and will be removed in 4.0. Use EventDispatcher with closure factories instead: 5x
    5x in Application::run from Codeception
```
> Provide test source code if related

```php
class CaptchaCest
{

    public function redirectsWhenCaptchaCorrect(FunctionalTester $i)
    {
        $url = '/sf/captcha/verify?rd=asdqwe';
        $i->amAuthenticatedAs('admin', ['ROLE_ADMIN']);
        $i->amOnPage('/sf/captcha/verify?rd=' . urlencode($url));
        $i->click('#recaptcha_submit');
        $i->seeCurrentUrlMatches('/\/sf\/captcha\/verify\?rd\=asdqwe/');
    }

    public function returnsBadRequestWhenNoRedirectUrlIsSend(FunctionalTester $i)
    {
        $i->amOnPage('/sf/captcha/verify');
        $i->seeResponseCodeIs(Response::HTTP_BAD_REQUEST);
    }

}
```
I've also added method in FunctionalTester for authenticating user with Symfony security system

```php
class FunctionalTester extends \Codeception\Actor
{
    use _generated\FunctionalTesterActions;

    public function amAuthenticatedAs($username, array $roles)
    {
        /** @var ContainerInterface $container */
        $token = $this->grabService('app.jwtservice')->createUserToken($username, $roles);

        $cookie = new Cookie(JWTService::COOKIE_KEY, $token);
        $this->grabService('test.client.cookiejar')->set($cookie);
        $this->setCookie($cookie->getName(), $cookie->getValue());
    }

}

```

### Details

* Codeception version: Codeception PHP Testing Framework v2.3.3
* PHP Version:PHP 5.6.30-0+deb8u1
* Operating System: Linux (docker)
* Installation type:  Composer
* List of installed packages (`composer show`)
* Suite configuration:

```yml
class_name: FunctionalTester
modules:
    enabled:
        - Symfony2:
            app_path: '../../app'
            var_path: '../../var'
        - \AppBundle\Helper\Functional```

**Comment by Naktibalda** [comment]
Duplicate of #4306
It is fixed in 2.3 branch already, wait for the next release (probably it will be made in 3-4 weeks time).

**Comment by mattwoo** [comment]
Sorry for duplicate, thanks for reply
